lib/controller/validation.js

```diff
@@ -0,0 +1,27 @@
+const prefsOutput = require('../cli/output/prefs');
+
+const supportedServers = ['ms', 'gitlab'];
+const supportedCommand = ['show', 'changelang', 'changeserver', 'logger'];
+
+const prefs = (args, options) => {
```

Comment:
We can ponder a bit on appropriate guidelines for the use of arrow functions in this project. If you can read a bit, and create a wiki page while also quoting good quality references, we can stick to the created guidelines everywhere in the project.

Reply:
Ok sir. I'll read up about it.